packages/react-qr/src/util.ts

```diff
@@ -53,9 +51,11 @@ export function createFrames (input: Uint8Array): Uint8Array[] {
     idx += FRAME_SIZE;
   }
 
+  const isMultipart = frames.length > 1 ? new Uint8Array([1]) : new Uint8Array([0]);
```

Comment:
Is this correct? Just looking at the . constant removed above - `const MULTIPART = new Uint8Array([0]);` - which is now set to `[1]`. (On non-multipart, we don't add the index and length as I recall)

Maybe just missing something, it has been a while...

Reply:
Yes, not convinced here - https://github.com/maciejhirsz/uos#payload-step

Reply:
FWICT it has always been adding the frame index and length, and I will go out on a limb here and say the spec is incorrect because the 5 bytes before `part_data` are what is constructed from the hot wallet and there is no sane way to set the first byte of each `part_data` with the multipart indicator `00` prior to `createFrames()`.

Example Full Raw Data from non-multipart messages from ReactNativeQR For Reference
```
  --- We Strip the first 3 bytes
  4 // indicates binary
  37 // indicates data length
  --- UOS Specific Data
  00 + // is not multipart
  0001 + // frame count
  0000 +  // which frame are we on?
  53 // indicates payload network
```

Reply:
So `00` is multipart as per https://github.com/maciejhirsz/uos#payload-step - the first entry (`01` is an extension range) Frame count of `0001` looks correct, we have a single frame. And the index of `0000` is also correct, since it is the first.

So for simplicity, it always send multipart, even if a single frame. If you have multiples, it would be `0x53` as the first byte. (As per spec)

Reply:
Ahh ok, I see that for simplicity you included the frame info for single frame data as well. Should this be changed to fit the spec?

If not, I can just revert this part back as it was, then check the frame length, i.e. is it multipart, from the Cold Signer instead.

Reply:
I really didn't see the need to explicitly handle 2 forms based of generation (and I still don't) - the multipart indicator works perfectly with a single frame with the counter and index.  It is not like it is a lot of code, but every bit of complexity saving helps.

(Sadly when parsing, you need to be able to handle both single and multiparts since you have no idea who is sending the info across, so the sender may have logic that generates a `0x53` or `0x00` payload depending on size)